### DISCUSSION TITLE
Expand Spend Management to Enterprise

### DISCUSSION THREAD

**Discussion by geoffg-sentry** [vercel/community#6838] [Idea]
Currently Spend Management is available only for Pro Plans and Enterprise users cannot benefit. Expand Spend Management feature to Enterprise users as well. 

[1] https://vercel.com/docs/pricing/spend-management

**Comment by leerob** [comment]
Hey! Would love to hear more. Are you looking for soft limits around certain price points or usage, or hard limits where you could pause projects?

**Reply by geoffg-sentry** [reply]
So limits so we can alert to increases in project spend and/or specific cost increases across features.
